presto-docs/src/main/sphinx/release/release-0.217.rst

```diff
@@ -0,0 +1,57 @@
+=============
+Release 0.217
+=============
+
+General Changes
+---------------
+
+* Fix poor query planning performance for queries that are submitted when the cluster does not have the minimum required number of workers available.
+* Fix an issue causing a union of a table with a ``VALUES`` statement to execute on a single machine, which may result in out of memory errors.
+* Improve the logical plan by removing unnecessary remote exchanges before window functions when the data is already available locally.
+* Add grouped execution support for window functions.
+* Add support for the ``BOOLEAN`` type for ``EXPLAIN IO`` statements.
+* Add support for choosing distribution type for semi joins based on the cost model when ``join_distribution_type='AUTOMATIC'``.
+* Add support for collecting table statistics on demand with the ``ANALYZE`` statement.
+* Add a config option (``query.stage-count-warning-threshold``) to specify a per-query threshold for the number of stages.
+  When this threshold is exceeded a ``TOO_MANY_STAGES`` warning is raised as part of the query information.
```

Comment:
@elonazoulay What is `query information`?

Reply:
I am not sure about the second part of the sentence:) Can it be changed to "When this threshold is exceeded a ``TOO_MANY_STAGES`` warning is raised" ?
Can I change it myself?

Reply:
Yes, that phrasing works better.